drivers/dma/Kconfig.iproc_pax

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig DMA_IPROC_PAX
+	bool
+	prompt "Broadcom PAX(PCIE<->AXI) DMA driver"
+	depends on PCIE_IPROC
+	default n
+
+menuconfig DMA_IPROC_PAX_V2
+	bool
+	prompt "Broadcom PAX(PCIE<->AXI) DMA driver version 2"
+	depends on PCIE_IPROC
+	default n
```

Comment:
If you make DMA_IPROC_PAX_V2 a sub-option, and make it depend on DMA_IPROC_PAX, you will only need one Cmake entry, and you can make the options below depend on DMA_IPROC_PAX instead of doing an if/endif section.

Reply:
Didn't fully understand your comment. As explained above DMA_IPROC_PAX_V2 is defined to address V2 hardware and DMA_IPROC_PAX is defined to address V1. So having 2 CMake entry is needed to differentiate.